reconcile/gitlab_labeler.py

```diff
@@ -10,6 +11,48 @@
 QONTRACT_INTEGRATION = 'gitlab-labeler'
 
 
+def guess_onboarding_status(changed_paths: List[str]) -> Optional[str]:
+    """
+    Guess the onboarding status of a given MR from the changed paths of the
+    MR.
+    """
+    apps = queries.get_apps()
+    #   logging.info(apps)
+    app_name = get_app_name(changed_paths)
+    if app_name is not None:
+        app = None
+        for a in apps:
+            if a['name'] == app_name:
+                app = a
+
+    if app is not None:
+        logging.info(app['onboardingStatus'])
+        return app['onboardingStatus']
+
+    return None
+
+
+def get_app_name(changed_paths: List[str]) -> Optional[str]:
```

Comment:
`Iterable[str]` is more general here and appropriate for cases where you're just looping through a list, tuple, set, etc. See [this doc](https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#abstract-collection-types) for more info. For method parameters, concrete type usage is generally reserved for cases where you rely on the actual `list` implementation. Using concrete types is fine for method returns types.